app/assets/javascripts/client/about/AboutSidebarCtrl.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('AboutSidebarCtrl', AboutSidebarCtrl);
+
+  AboutSidebarCtrl.$inject = [
+    '$scope',
+    '$rootScope',
+    '$element',
+    '$location',
+    '$timeout',
+    '$modal'
+  ];
+
+  function AboutSidebarCtrl($scope, $rootScope, $element, $location, $timeout, $modal) {
+    var vm = this;
+
+    new Clipboard("[data-clipboard-target]");
+
+    vm.openAssessmentModal = function() {
+      $modal.open({
```

Comment:
You can't open more than one modal at the same time. After opening a modal, attempting to click a trigger for another one will just close the first.

Very good question re authentication. I guess I'll defer to @candyj0311